src/components/Task/Modal/Form/Complete/TaskModalCompleteForm.tsx

```diff
@@ -222,6 +222,10 @@ const TaskModalCompleteForm = ({
       openTaskModal({
         view: TaskModalEnum.Add,
         defaultValues: {
+          subject:
```

Comment:
This works as expected when I tried it. I wonder if you should also add this feature when logging a task.

I think some people will want it to keep the previous task name even if it is a different task action. I wonder if we could have it look at the previous task action, check if it was customized, and then if it was customized, keep the old one even if the action is different.

Reply:
^ I agree, we should add this functionality to logs task.

Reply:
> I wonder if we could have it look at the previous task action, check if it was customized, and then if it was customized, keep the old one even if the action is different.

You saying that made me realize that there is an easy way to check whether the task subject was customized: just check whether it matches the generated task subject or not. This logic will break if the task subjects change on the server, but I think this is OK because that should be fairly rare.